openasip/src/base/osal/InstructionExecutor.cc

```diff
@@ -0,0 +1,287 @@
+#include <string>
```

Comment:
License/copyrights. Also the name of the file could be rethought. RISCVInstructionExecutor?